### PR TITLE
Cleanup readme for MSAL.js and MSAL Node, Adding initial Auth Code Sample

### PR DESCRIPTION
**Pull request by mvrak**
Auth Code Sample needs more work to:

- demonstrate how auth code is added to existing app
- include some web ux to make flow clearer

For now the readme's are in a good situation

Had an issue where I had to install uuid to run the auth code sample, even though it is specified in the package configuration.  Will have to test that some more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,23 @@
 # Microsoft Authentication Library for JavaScript (MSAL.js)
 
-The Microsoft Authentication Library for JavaScript enables client-side JavaScript web applications, running in a web browser, to authenticate users using [Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview) for work and school accounts (AAD), Microsoft personal accounts (MSA), and social identity providers like Facebook, Google, LinkedIn, Microsoft accounts, etc. through [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. It also enables your app to get tokens to access [Microsoft Cloud](https://www.microsoft.com/enterprise) services such as [Microsoft Graph](https://graph.microsoft.io).
+The Microsoft Authentication Library for JavaScript adds Auth to Javascript applications, and access to resource APIs in the [Microsoft Cloud](https://www.microsoft.com/enterprise) such as [Microsoft Graph](https://graph.microsoft.io).
 
-## Repository
+MSAL.js now supports Node.js and Electron!
```
Comment (jasonnutter) [inline README.md:5]: I think references to Node/Electron should go in the MSAL Node readme, and should make it clear what Electron support means (there are different scenarios and I don't believe all of them are supported yet).

File: README.md [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
-### Core and wrapper libaries
+MSAL.js supports both consumer and enterprise Microsoft accounts.  This can be restricted if desired.  Only applications that use special consumer or enterprise data will need to differentiate.
 
-The [`lib`](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib) folder contains the source code for all of our libraries. You will also find all the details about **installing the libraries**, in their respective README.md.
+MSAL.js supports social identity providers like Facebook, Google, and LinkedIn through the [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. 
+
+## How do I get Started?
+
+Follow the link that matches your app:
+- [Traditional web app](lib/msal-browser/)
```
Comment (jasonnutter) [inline README.md:14]: Do we want to say "single-page applications"? To align with how it is labeled in teh portal. Traditional web app can mean a server-side app in our nomenclature.

File: README.md (hunk 1 above)
Comment (pkanher617) [reply README.md:5]: +1

File: lib/msal-node/README.md [hunk 3]
```diff
@@ -8,21 +8,25 @@
...
-4. [Prerequisites](#prerequisites)
-5. [Installation](#installation)
-6. [Usage](#usage)
-7. [Samples](#samples)
-8. [Build Library](#build-and-test)
-9. [Security Reporting](#security-reporting)
-10. [License](#license)
-11. [Code of Conduct](#we-value-and-adhere-to-the-microsoft-open-source-code-of-conduct)
+1. [FAQ](#faq)
+1. [Prerequisites](#prerequisites)
```
Comment (pkanher617) [inline lib/msal-node/README.md:12]: Link to releases should be added back in.

File: README.md (hunk 2 above)
Comment (pkanher617) [reply README.md:14]: +1

File: lib/msal-node/README.md [hunk 4]
```diff
@@ -61,15 +65,12 @@ More details on scenarios and the authentication flows that map to each of them
 
 See [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md).
 
-## Prerequisites
-
-Before using `@azure/msal-node` you will need to register your app in the azure portal:  
-
-- [App registration](https://docs.microsoft.com/en-us/graph/auth-register-app-v2)
-
```
Comment (pkanher617) [inline lib/msal-node/README.md:69]: We need to keep these pre-requisites, especially for apps using MSAL.js 2.x

File: lib/msal-node/README.md [hunk 5]
```diff
@@ -94,30 +95,6 @@ There are multiple [samples](https://github.com/AzureAD/microsoft-authentication
...
-// To run build only for node package
-npm run build
-
-// Change to the msal-node package directory
-cd lib/msal-node/
-
-// To run build only for node package
-npm run build
-```
-
```
Comment (pkanher617) [inline lib/msal-node/README.md:120]: This can be added back for now, I will update the contributing.md file and remove this in a future PR.

**Review by pkanher617 — CHANGES_REQUESTED** [review]
A few changes

**Pull request closed without merging**